_i18n/es.yml

```diff
@@ -218,6 +218,7 @@ clients-partners:
    skillsmatter_description: Skills Matter, con más de 70.000 miembros, fue fundada para ayudar a impulsar la innovación en la ingeniería de software reuniendo a desarrolladores inteligentes y creativos para aprender y compartir nuevas habilidades, descubrir nuevas tecnologías y evolucionar las prácticas e ideas. Hemos sido socios con Skills Matter desde nuestros inicios. Juntos, organizamos eventos para la comunidad, así como cursos de formación públicos y privados.
    jetbrains_description: Jetbrains es conocido por sus herramientas de desarrollo de software. Ayudamos a nuestros clientes a sacar el máximo partido de estas herramientas, apoyando las prácticas XP y la calidad de software. Somos expertos en sus herramientas, tales como Team City, ReSharper, e IntelliJ IDEA. También ayudamos a nuestros clientes a adoptar el lenguaje de programación Kotlin.
    github_description: Github es una conocida plataforma de hosting de proyectos online. Entendemos que utilizar un repositorio de código distribuido como Git no es sólo una cuestión de habilidades técnicas, sino también un cambio de paradigma en cómo colaboran los equipos de desarrollo de software. Ayudamos a nuestros clientes a adoptar Github y Github Enterprise.
+   satalia_description: Satalia resuelve problemas complejos utilizando la ciencia avanzada de datos, optimización y tecnologías de inteligencia artificial. Construyen soluciones completas a gran escala en areas que incluyen rutas, horarios, planificación y verificatión. Reconocidos líderes en IA, Satalia ha sido galardonada con el Gartner Cool Vendor 2016. 
```

Comment:
A couple of typos:
**á**reas
verifica**c**ión
And a suggestion:
What about "utilizando ciencia de datos avanzada" instead of "utilizando la ciencia avanzada de datos"?